manifests/init.pp

```diff
@@ -91,19 +131,29 @@
 #
 # @example
 #   include pulpcore
+#
+# @see https://docs.djangoproject.com/en/2.2/howto/static-files/
 class pulpcore (
-  Stdlib::Absolutepath $cache_dir = '/var/lib/pulp/tmp',
-  Stdlib::Absolutepath $config_dir = '/etc/pulp',
   String $user = 'pulp',
   String $group = 'pulp',
   Stdlib::Absolutepath $user_home = '/var/lib/pulp',
-  Boolean $manage_apache = true,
+  Stdlib::Absolutepath $config_dir = '/etc/pulp',
+  Stdlib::Absolutepath $cache_dir = '/var/lib/pulp/tmp',
+  Stdlib::Absolutepath $media_root = '/var/lib/pulp/media',
+  Stdlib::Absolutepath $static_root = '/var/lib/pulp/assets',
```

Comment:
SELinux prevents Apache from serving this directory since it has the content type `pulpcore_var_lib_t`. Going to change it `static` just to continue.

Reply:
Given the intent of the module is to primarily support the `foreman-installer` use case, *and* we have a desire to expose this module as a top level module, what is the value in exposing these directories as primary inputs the module that a user could configure?